### DISCUSSION TITLE
Unprofessional screen on github landing page (guest page)

### DISCUSSION THREAD

**Discussion by wildpin** [community/community#14091] [Other Feature Feedback, Questions, & Ideas]
I just spotted some mistakes in github landing page under "Give your code a home in the cloud" section.

The browser mockup has 2 arrows pointing to right, and the JS colour (blue, in this screen) from sample repo doesn't match the official colour. 😅

![Screenshot 2022-04-02 at 14 35 00](https://user-images.githubusercontent.com/70672233/161381575-f0a62f24-8af9-4222-99b4-2f7a25593361.png)

**Comment by georgelemon** [comment]
Up up
